### PR TITLE
Translations/r20.22.2v2

### PR DESCRIPTION
**Pull request by johngwilkinson**

### CODE REVIEW

**Comment by johngwilkinson** [comment]
Translations returned `cy.js` instead of `cy-gb.js`. This is apparently an error, so I renamed it: https://d2l.slack.com/archives/C0TF4KSGZ/p1643312307094700

**Comment by johngwilkinson** [comment]
Sorry, this branch is mis-named: it is the translations as applied to master, not r20.22.2

File: locales/cy-gb.js [hunk 1]
```diff
@@ -23,9 +23,9 @@ export default {
 	"dashboard:quizDetailsNumberOfLearnerAnswers": "Nifer o Atebion Dysgwyr",
 	"dashboard:quizDetailsQuizId": "Rhif adnabod y cwis:",
 	"dashboard:quizDetailsReliabilityCoefficientTooltip": "Cyfrifir cysondeb ar gyfer pob cwestiwn p’un a yw cwestiynau'r cwis i gyd yn ddwyrannol ai peidio. Fel arfer bydd cysondeb yn amrywio o 0 i 1 gyda’r posibilrwydd o werthoedd negyddol.",
-	"dashboard:quizDetailsFlaggedQuizItemsTooltip": 'Eitemau Cwis Wedi’u Hamlygu',
-	"dashboard:quizDetailsCourseNameTemplate": '{courseName} (rhif adnabod: {courseId})',
-	"dashboard:quizDetailsFlaggedRow": 'Mae gan y cwis hwn un neu fwy o fetrigau y gallai fod angen eu hadolygu.',
+	"dashboard:quizDetailsFlaggedQuizItemsTooltip": "Nifer yr eitemau cwis sy'n dangos pwynt data negyddol neu is na'r trothwy a ddiffinnir yn y Gosodiadau.",
+	"dashboard:quizDetailsCourseNameTemplate": "{courseName} (rhif adnabod: {courseId})",
+	"dashboard:quizDetailsFlaggedRow": "Mae gan y cwis hwn un neu fwy o fetrigau y gallai fod angen eu hadolygu.",
```
Comment (anhill-D2L) [inline locales/cy-gb.js:28]: Nice to see us switching from single to double quotes for consistency

**Review by NicholasHallman — APPROVED** [review]

**Review by anhill-D2L — APPROVED** [review]

**Comment by anhill-D2L** [comment]
I feel like a bot would provide just as good, or better reviews on translation PR's than myself.

File: locales/fr-ON.js [hunk 2]
```diff
@@ -92,14 +98,19 @@ export default {
...
 	"quizItems:na": 'S.O.',
+	"quizItems:inQuestionPoolTrue": "Oui",
+	"quizItems:inQuestionPoolFalse": "Non",
 	"quizItems:summaryHeading": 'Sommaire',
 	"quizItems:lowMeanHeading": 'Low Mean',
 	"quizItems:lowMeanMessage": 'Quiz items with a mean of {score} or lower.',
-
+	"quizItems:weakCorrelationTitle": "Faible corrélation",
+	"quizItems:weakCorrelationDescription": "Éléments du questionnaire dont la corrélation est de {num} ou moins.",
 	"quizItems:highDiscriminationCardTitle": "High Discrimination",
```
Comment (johngwilkinson) [inline locales/fr-ON.js:109]: Ugh, some of this stuff was translated before, but I think it only made it into the release branch. I'll add another commit here to bring those changes over once I finish merging over there.

**Pull request merged**